frontend/__mocks__/k8sResourcesMocks.ts

```diff
@@ -9,6 +9,264 @@ export const testNamespace: K8sResourceKind = {
   },
 };
 
+export const testPodInstance: PodKind = {
+  kind: 'Pod',
```

Comment:
We dont need all the pod fields to be present, lets stick to the minimum.
Lets remove:
- managedFields
-  spec.volumes
-  spec.toletations

Reply:
Ok, I've removed the fields.